gear/push_template/src/python/template_app/run.py

```diff
@@ -49,19 +53,34 @@ def create(
         api_key = get_api_key(context)
         client.set_fw_client(
             FWClient(api_key=api_key, client_name="push-template"))
+        group_id = context.config.get("template_group")
+        if not group_id:
+            raise GearExecutionError("Expected \"template_group\"")
+        template_label = context.config.get("template_project")
+        if not template_label:
+            raise GearExecutionError("Expected \"template_label\"")
 
         return TemplatingVisitor(
             admin_id=context.config.get("admin_group", "nacc"),
             client=client,
+            template_group=group_id,
+            template_label=template_label,
             new_only=context.config.get("new_only", False))
 
     def run(self, context: GearToolkitContext) -> None:
-        template_map = get_template_projects(group=self.admin_group(
```

Comment:
Can remove the import for this, but also curious if there's a need for this specific method/file now that you've moved the pattern finding to the `TemplateProject` class. It's fine as it's "common" code but afaik it isn't being used anywhere else.

Reply:
Would be fine to remove it